docs/README.md

```diff
@@ -60,3 +60,16 @@ These files are called checkpoints and they save information about deployed asse
 As they are checked-in into Version Control System you can use them to track the deployed state of your assets.
 
 Read more about checkpoints [here](/docs/execution-checkpoints.md).
+
+### Typescript Projects
+
+`Typescript` project requires a compilation to `js` files. So that algob can execute `js` files.
+To develop in `typescript`, please remember these points:
+
+- Make sure to compile your `ts` project using `tsc --build .` Never forget to compile files.
+- If you are actively developing, please use `tsc -w -p .` to build your files in real time.
+- Best way to build project is to run tsc in a watch mode.
```

Comment:
Is this different from the above point?

Reply:
yes, it says the below method is better

Reply:
We could just add `(this is recommended)` in the above point. Seems more intuitive to me

Reply:
Fair point.